demos/exchange/full-demo.sh

```
@@ -0,0 +1,111 @@
+#!/bin/sh
+
+################################################################################
+
+source .local.env
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)